internal/server/server.go

```diff
@@ -171,10 +170,10 @@ func Start(ctx context.Context) error {
 		if err := rt.LoadPolicies(rps); err != nil {
 			return fmt.Errorf("failed to load policies into rule table: %w", err)
 		}
+	}
 
-		if ss, ok := store.(storage.Subscribable); ok {
-			ss.Subscribe(rt)
-		}
+	if ss, ok := policyLoader.(storage.Subscribable); ok {
+		ss.Subscribe(rt)
```

Comment:
https://github.com/cerbos/cerbos/issues/2451#issuecomment-2601702657

> I'm not sure I understand why this is the case. For mutable stores, the original stores remain as the source of truth, and a "partial rule table" is generated at query time (and discarded afterwards).

In the mutable-store case, partial rule tables are (currently) generated from the mutable store per-query and discarded afterwards. I chose to do this as an initial step as it seemed odd to maintain two sources of mutable truth.

This may or may not be related to the recent [E2E test failure](https://github.com/cerbos/cerbos/actions/runs/12859261034/job/35849465024). I'm looking in to that.